src/core/include/openvino/core/function.hpp

```diff
@@ -353,4 +353,43 @@ class OPENVINO_API AttributeAdapter<std::shared_ptr<ov::Function>>
     OPENVINO_RTTI("AttributeAdapter<std::shared_ptr<Function>");
     BWDCMP_RTTI_DECLARATION;
 };
+
+namespace util {
```

Comment:
why is it `util`?

Reply:
This was from original PR #8560 . @ilyachur can you please comment?

Reply:
changed to 'pass'